RELEASE.md

```diff
@@ -0,0 +1,118 @@
+# Releasing KubeArchive
+
+## Requirements/Assumptions
+
+* Using the release-notes tool that K8s uses
+* Using `kind/breaking`, `kind/documentation`, `kind/bug` and `kind/feature` labels
+* Using a code markdown block with release notes:
+    ```release-note
+    Something that will be added to the top of the release notes.
+    ```
+    or
+    ```release-note
+    NONE
+    ```
+* Chart version will be manually increased or will be the same as the application
+version to simplifiy the process.
+* No changelog, we can link to the branch comparison on GitHub, which will correspond
```

Comment:
If we do this I think we need to have a more standard process for the commits and PRs descriptions. I would like to defined a standard way to easily identify if the commit is a fix, a feature...

Reply:
In this case we will be using `kind/breaking`, `kind/documentation`, `kind/bug` and `kind/feature` labels